samples/net/mqtt/doc/description.rst

```diff
@@ -275,7 +275,7 @@ Troubleshooting
 To enable more verbose logging from the MQTT helper library, enable the :kconfig:option:`CONFIG_MQTT_HELPER_LOG_LEVEL_DBG` option.
 
 * If you are having issues with connectivity on nRF91 Series devices, see the `Trace Collector`_ documentation to learn how to capture modem traces in order to debug network traffic in Wireshark.
-  The sample enables modem traces by default, as set by the :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE` option.
+  The sample enables modem traces by default.
```

Comment:
Shouldn't we mention about the nrf91-modem-trace-uart snippet in this case?

Reply:
We're not adding the snippet in this case, but adding the config to the sample itself. I've updated the release notes to reflect this.